python/graphstorm/run/gsgnn_lp/gsgnn_lp.py

```diff
@@ -88,7 +88,8 @@ def main(config_args):
     config = GSConfig(config_args)
     config.verify_arguments(True)
 
-    gs.initialize(ip_config=config.ip_config, backend=config.backend)
+    gs.initialize(ip_config=config.ip_config, backend=config.backend,
```

Comment:
What about gsgnn_np and gsgnn_ep?

Reply:
We don't support `gsgnn_ep` now. Added TODO. Fixed np.

Reply:
So do we need to change gsgnn_np?